backend/src/main/java/de/jmpsoftware/backend/service/ArtNetThread.java

```diff
@@ -104,7 +105,7 @@ boolean checkPipeReadDoneCorrect(int pipeReadReturn) throws IOException {
             byte[] buffer = new byte[100];
             while (pipe.available() > 0)
             {
-                int trashIt = pipe.read(buffer,0,100);
+                @SuppressWarnings("unused") int trashIt = pipe.read(buffer,0,100);
```

Comment:
Why is `trashIt` declared here?

Consider using the return value (how many bytes have been read) as condition to exit the loop. If less than buffer-size has been read the stream is usually done sending data.

Reply:
Das ist die Notfall-Funktion wenn die Pipe Daten Falsch sein könnten. Ich lese hier nur den kompletten Pipe aus damit ich